randovania/games/prime1/presets/moderate_challenge.rdvpreset

```diff
@@ -236,6 +236,7 @@
         "small_samus": false,
         "shuffle_item_pos": false,
         "items_every_room": false,
+        "spring_ball": false,
```

Comment:
Rather than update the presets directly, implement a preset migration for this change (thus maintaining forward compatibility).